lib/cc/service/http.rb

```diff
@@ -15,6 +15,34 @@ def default_http_options
     end
   end
 
+  def post(url, body = nil, headers = nil, &block)
+    begin
+      block ||= lambda{|*args| Hash.new }
+      response = http_post(url, body, headers)
+      {
+        ok: response.success?,
+        params: body.as_json,
+        endpoint_url: url,
+        http_status: response.status
+      }.merge(block.call(response))
+    rescue CC::Service::HTTPError => e
```

Comment:
That's true, but I think separating the exceptions is more appropriate and clearer. That the return values happen to be similar is not a reason to introduce a less clear but of code, IMHO.